### DISCUSSION TITLE
Codewars App

### DISCUSSION THREAD

**Discussion by gfreundt** [codewars/codewars.com#2730] [Ideas]
Hello,

Are there plans for a Codewars app? I would definitely use it...

My technical skills don't include app development or I would give it a go.

Thanks,

Gabriel

**Comment by Danilo0528** [comment]
Yo igual no tengo conocimientos en desarrollo de apps
pero me gustaria tambien la usaria

**Comment by nazmus-ashrafi** [comment]
That's a great idea, I would use it too.

**Comment by umidevopz** [comment]
**yeah boys, we need app for both android and ios**

**Comment by televisionia** [comment]
Yeah, an app would be insane.

**Comment by DeraCodings** [comment]
Sure, we could do with the app.
Maybe we can start contributing; though I am not an app developer but I'm sure I can help. 
Or make it a PWA

**Comment by logical-luke** [comment]
IMO it might be an inconvenient experience to code on a small device like a mobile phone.
Should an app provide the same features as the website?

**Reply by televisionia** [reply]
There are some people out there that legitimately code on their phone, but even some crazy people use terminals on android phones.

**Comment by satyamnamnu** [comment]
I wish i had that skill to create apps :((
